pkg/inspector/image-inspector.go

```diff
@@ -499,27 +526,28 @@ func (i *defaultImageInspector) getAuthConfigs() (*docker.AuthConfigurations, er
 	return imagePullAuths, nil
 }
 
-func (i *defaultImageInspector) scanImage(s iiapi.Scanner) ([]byte, []byte, error) {
+// TODO: Move this into openscap package
```

Comment:
so I will do this refactor now, then use some `tig` magic to reduce scope for this PR so we can get this in smaller chunks hopefully.